projects/wallet-template/src/containers/WalletPopup/pages/Welcome.tsx

```diff
@@ -1,101 +1,86 @@
-  const onSubmit: SubmitHandler<FormFields> = async ({ password }) => {
-    await rpc.client.createPassword(password)
-    await refresh()
-    reset()
-  }
+  const navigate = useNavigate()
+
   return (
-    <Layout>
-      {isSubmitSuccessful && <Navigate to="/accounts" replace={true} />}
-      <div className="my-4">
-        <h1 className="text-3xl font-bold text-center">Welcome</h1>
-      </div>
-      <div className="mt-12 mb-4">
-        <h1 className="text-xl font-bold text-center">Create password</h1>
-      </div>
-      <form
-        onSubmit={(e) =>
-          handleSubmit(onSubmit)(e).catch((e) =>
-            setError("root", {
-              message: e.message ?? "Error creating password",
-            }),
-          )
-        }
-      >
-        <div className="my-4">
-          <input
-            type="password"
-            placeholder="Enter password"
-            className="w-full p-3 text-lg border rounded"
-            disabled={isSubmitting}
-            autoFocus
-            {...register("password", {
-              required: "You must specify a password",
-              minLength: {
-                value: 6,
-                message: "Password must have at least 6 characters",
-              },
-            })}
-          />
-          {errors.password && (
-            <div className="text-red-500">{errors.password.message}</div>
-          )}
-        </div>
-        <div className="my-4">
-          <input
-            type="password"
-            placeholder="Confirm password"
-            className="w-full p-3 text-lg border rounded"
-            disabled={isSubmitting}
-            {...register("passwordConfirm", {
-              required: "You must confirm the password",
-              validate: (value) =>
-                value === watch("password") || "The passwords do not match",
-            })}
-          />
-          {errors.passwordConfirm && (
-            <div className="text-red-500">{errors.passwordConfirm.message}</div>
-          )}
-        </div>
-        <div className="my-4">
-          <button
-            type="submit"
-            disabled={isSubmitting}
-            className="w-full p-3 text-lg rounded border border-[#24cc85] text-[#24cc85] hover:text-white hover:bg-[#24cc85]"
+    <Layout2>
+      <Card className="flex-grow w-full max-w-md min-h-full rounded-none lg:rounded">
+        <CardHeader className="text-center">
+          <div className="relative text-6xl font-bold text-teal-400">_</div>
+          <CardTitle className="mt-6 text-2xl font-extrabold">
+            substrate
+            <br />
+            <span className="text-5xl text-primary">Connect</span>
+          </CardTitle>
+          <CardDescription className="mt-2">
+            The easiest way to connect to Polkadot, Kusama, and Substrate-based
+            chains with a light client
+          </CardDescription>
+        </CardHeader>
+        {/* TODO: Replace the filler text below with something real */}
```

Comment:
is this still TODO?

Reply:
Yeah, unless we just want to go with that text. probably something more light client related